src/html.rs

```diff
@@ -9,7 +9,8 @@ use http::retrieve_asset;
 use js::attr_is_event_handler;
 use std::collections::HashMap;
 use std::default::Default;
-use utils::{data_to_dataurl, is_valid_url, resolve_url, url_has_protocol};
+use std::fmt::Write as OtherWrite;
```

Comment:
Oh yup, you're right.  I had that in there when I was doing a bit of debug work, and I needed to import a couple different `Write`s, but I've since taken out all that code.  This must just be a straggler.